src/js/core/methods/helpers/signtxVerify.js

```diff
@@ -68,18 +68,29 @@ const deriveBech32Output = (program: Buffer): Buffer => {
     return scriptSig;
 };
 
-const deriveOutputScript = async (getHDNode: GetHDNode, output: TransactionOutput, coinInfo: BitcoinNetworkInfo): Promise<Buffer> => {
+const deriveOutputScript = async (getHDNode: GetHDNode, output: TransactionOutput, coinInfo: BitcoinNetworkInfo): Promise<Buffer | void> => {
     if (output.op_return_data) {
         return BitcoinJsScript.nullData.output.encode(Buffer.from(output.op_return_data, 'hex'));
     }
     if (!output.address_n && !output.address) {
         throw ERRORS.TypedError('Runtime', 'deriveOutputScript: Neither address or address_n is set');
     }
 
+    // compatibility for Casa - If passing in multisig params with p2shw, skip validation
+    if (output.multisig && output.script_type === 'PAYTOP2SHWITNESS') {
```

Comment:
i would leave `if (output.multisig) return;`
like i said, script_type is irrelevant at this point

also check at line 90 could be removed

i'll do it